### PR TITLE
mrp embed slide view 

### PR DESCRIPTION
**Pull request by jpr-odoo**
Description of the issue/feature this PR addresses:
task: https://www.odoo.com/web?#id=1909010&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/js/fields/basic_fields.js [hunk 1]
```diff
@@ -1610,6 +1610,124 @@ var CharCopyClipboard = FieldChar.extend(CopyClipboard, {
...
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * initialize an iframe for viewer
+     *
+     * @private
+     */
```
Comment (svs-odoo) [inline addons/web/static/src/js/fields/basic_fields.js]: Don't forget to specify what your method returns.
```javascript
     /**
     * Initializes and returns an iframe for the viewer
     *
     * @private
     * @returns {jQueryElement}
     */
```

File: addons/web/static/src/js/fields/basic_fields.js [hunk 2]
```diff
@@ -1610,6 +1610,124 @@ var CharCopyClipboard = FieldChar.extend(CopyClipboard, {
...
+            this.$el.addClass('o_embed_url_viewer');
+            this.$el.append(this._prepareIframe());
+        }
+        this._updateIframePreview();
+    },
+    /**
+     * set the associated src for embed iframe viewer
+     *
+     * @private
+     */
```
Comment (svs-odoo) [inline addons/web/static/src/js/fields/basic_fields.js]: Same here, don't forget to specify what your method returns.
```javascript
     /**
     * Set the associated src for embed iframe viewer
     *
     * @private
     * @returns {string} source of the google slide
     */
```

**Comment by sle-odoo** [comment]
robodoo r+

**Comment by sle-odoo** [comment]
robodoo r+

**Pull request closed without merging**